sycl/include/sycl/ext/oneapi/experimental/reduction_properties.hpp

```diff
@@ -0,0 +1,145 @@
+//==------- properties.hpp - SYCL properties associated with reductions ----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+#define SYCL_EXT_ONEAPI_REDUCTION_PROPERTIES
+
+#include <sycl/ext/oneapi/properties/property.hpp>
+#include <sycl/ext/oneapi/properties/property_value.hpp>
+#include <sycl/reduction.hpp>
+
+namespace sycl {
+inline namespace _V1 {
+namespace ext {
+namespace oneapi {
+namespace experimental {
+
+struct deterministic_key {
+  using value_t = property_value<deterministic_key>;
+};
+inline constexpr deterministic_key::value_t deterministic;
+
+struct initialize_to_identity_key {
+  using value_t = property_value<initialize_to_identity_key>;
+};
+inline constexpr initialize_to_identity_key::value_t initialize_to_identity;
+
+template <> struct is_property_key<deterministic_key> : std::true_type {};
+template <>
+struct is_property_key<initialize_to_identity_key> : std::true_type {};
```

Comment:
This isn't needed since about March, see https://github.com/intel/llvm/pull/12831. That said, I hope we'll change it even more soon: https://github.com/intel/llvm/pull/15752, but you can ignore that for now.

Reply:
Fixed in 427a98f4a3f285c945f1a51e2dae7341a30bce7a.